app/containers/Forms/NaicsField.tsx

```diff
@@ -0,0 +1,62 @@
+import {FieldProps} from '@rjsf/core';
+import React from 'react';
+import {Alert} from 'react-bootstrap';
+import {createFragmentContainer, graphql} from 'react-relay';
+import {NaicsField_query} from '__generated__/NaicsField_query.graphql';
+
+interface Props extends FieldProps<string> {
+  query: NaicsField_query;
+}
+
+export const NaicsFieldComponent: React.FunctionComponent<Props> = (props) => {
+  const naicsList = props.query.naicsCodes.edges.map(
+    (edge) => edge.node.naicsCode
+  );
+
+  const fieldProps = {
+    ...props,
+    schema: {
+      ...props.schema,
+      enum: naicsList,
+      enumNames: props.query.naicsCodes.edges.map((edge) => {
+        const naicsCode = edge.node;
+        return `${naicsCode.naicsCode} - ${naicsCode.ciipSector} - ${naicsCode.naicsDescription}`;
```

Comment:
```suggestion
        return `${naicsCode.naicsCode} - ${naicsCode.naicsDescription}`;
```

I think that the CIIP sector is more for internal use